src/components/LocationSearch.js

```diff
@@ -1,12 +1,44 @@
 import { useState } from 'react'
 import FormControl from '@mui/material/FormControl'
 import TextField from '@mui/material/TextField'
+import { Autocomplete} from '@react-google-maps/api';
 
-const LocationSearch = ({type, initialLocation, onLocationChange}) => {
+const LocationSearch = ({type, initialLocation, onLocationChange,initialLocationChange,sampleText}) => {
```

Comment:
rename initialLocationChange  as locationChange.

Reply:
also remove sampleText, replace with what it means. Here `type`?

Reply:
updated